plasma_framework/contracts/mocks/exits/payment/routers/PaymentInFlightExitRouterMock.sol

```diff
@@ -79,6 +89,32 @@ contract PaymentInFlightExitRouterMock is PaymentInFlightExitRouter {
         framework.flagOutputSpent(_outputId);
     }
 
+    /**
+     * This function helps test reentrant by making this function itself the first call with 'nonReentrant' protection
+     * So all other PaymentExitGame functions that is protected by 'nonReentrant' too would fail as it would be considered as re-entrancy
```

Comment:
So all other PaymentExitGame functions that are protected by 'nonReentrant' too would fail as it would be considered as re-entrancy